### ISSUE TITLE
Date Validation

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/kgtk#42]
The command `kgtk generate_wikidata_triples`  is dropping dates found on Wikidata.

This is the content of ignored.log file:
```
Corrupted statement at line number: 7145 with id  with current corrupted id None
Corrupted statement at line number: 7146 with id  with current corrupted id None
Corrupted statement at line number: 29804 with id  with current corrupted id None
Corrupted statement at line number: 32151 with id  with current corrupted id None
Corrupted statement at line number: 63640 with id  with current corrupted id None
```

The corresponding lines are:
```
Q201000001      P580    ^-4000-01-01T00:00:00Z/11
Q201000001      P582    ^49500-01-01T00:00:00Z/11
Q201000028      P582    ^5675674-01-01T00:00:00Z/11
Q201000031      P582    ^20080-01-01T00:00:00Z/9
Q201000054      P580    ^-0499-01-01T00:00:00Z/11
```

**Comment by rongpenl** [comment]
https://github.com/usc-isi-i2/etk/issues/414 Related issue.

**Comment by rongpenl** [comment]
Fixed with #61 .
